TEMPLATE.md

```diff
@@ -35,17 +39,9 @@ requires (*optional): <AIP number(s)>
  > What are we committing to not doing and why are they scoped out?
 
 ...
+# Proposed solution
 
-## Motivation
-
- > Describe the impetus for this change. What does it accomplish?
- > What might occur if we do not accept this proposal?
-
-...
-
-## Impact
-
- > Which audiences are impacted by this change? What type of action does the audience need to take?
+ > Define the straw person solution with enough details to make it clear why this is the preferred solution.
```

Comment:
Strawman... I don't think I've ever heard of strawperson being used before. I get the PC connotations, but they might actually confuse people.